hash_practice/exercises.py

```diff
@@ -5,15 +5,55 @@ def grouped_anagrams(strings):
         Time Complexity: ?
         Space Complexity: ?
     """

```

Comment:
👍  Time/space complexity?

I think this is an O(n^2) time complexity algorithm in that the set difference function is an O(n) method.